front/src/pods/master/master.component.tsx

```diff
@@ -180,6 +181,12 @@ const ShowVotingResultsComponent: React.FC<ShowVotingResultsProps> = props => {
         <div className={classes.containerComponent}>
           <TablePlayerComponent playersCollection={playerVotingStatus} />
         </div>
+        <div className={classes.containerComponent}>
+          <h2 className={classes.title}>Trend</h2>
```

Comment:
Why don't move this title inside `TablePlayerTrendComponent` as `TablePlayerComponent`? Please check `TablePlayerComponent` that use Typography and headingLevel prop to change heading components by props